assembl/models/path_utils.py

```diff
@@ -230,9 +233,13 @@ def base_query(labeled=False):
                 query = db.query(post.id.label("post_id"))
             else:
                 query = db.query(post.id)
-            query = query.join(
-                content, (content.id == post.id) &
-                         (content.tombstone_date == None))
+            query = query.join(content, content.id == post.id)
```

Comment:
documented in post.delete_post, and PostPath.as_clause_base. Tell me if intelligible at this point.